docs/sources/installation/security.md

```diff
@@ -12,29 +12,29 @@ weight = 2
 
 # Security
 
-## Data source proxy and protecting internal services
+If you run non-Grafana web services on your Grafana server or within its local network, then they might be vulnerable to exploitation through the Grafana data source proxy or other methods.
 
-If you have non-Grafana web services running on your Grafana server or within its local network, these may be vulnerable to exploitation via the Grafana data source proxy.
+To prevent this type of exploitation from happening, we recommend that you apply one or more of the precautions listed below.
 
-To prevent this type of exploitation from happening we explain a couple of different solutions  below.
+## Limit IP addresses/hostnames for data source URL
 
-### Configure Grafana to only allow certain IP addresses/hostnames to be used as data source url
+You can configure Grafana to only allow certain IP addresses or hostnames to be used as data source URL and by that proxied through the Grafana data source proxy. Refer to [data_source_proxy_whitelist]({{< relref "configuration/#data-source-proxy-whitelist" >}}) for usage instructions.
 
-You can configure Grafana to only allow certain IP addresses/hostnames to be used as data source url and by that proxied through the Grafana data source proxy. See [data_source_proxy_whitelist]({{< relref "configuration/#data-source-proxy-whitelist" >}}) for usage instructions.
+## Firewall rules
 
-### Firewall rules
+Configure a firewall to restrict Grafana from making network requests to certain internal web services. 
 
-You should be able to configure a firewall, for example using iptables, to restrict Grafana from making network requests to certain internal web services.
+There are many firewall tools available, refer to the documentation for your specific security tool. For example, Linux users can use [iptables](https://en.wikipedia.org/wiki/Iptables).
 
-### Proxy server
+## Proxy server
 
-You should be able to require all network requests being made by Grafana to go through a proxy server.
+Require all network requests being made by Grafana to go through a proxy server. Refer to [Running Grafana behind a reverse proxy](https://grafana.com/docs/grafana/latest/installation/behind_proxy/) for examples.
```

Comment:
This section is confusing using an outbound proxy for requests *from* Grafana (used for outbound requests from the datasource and plugin proxies, etc) with using a reverse proxy in front of grafana (which processes client requests).

Reply:
What needs to change in this passage?

Reply:
@oddlittlebird -- remove the sentance "Refer to [Running Grafana behind a reverse proxy](https://grafana.com/docs/grafana/latest/installation/behind_proxy/) for examples"  

Note -- that doc shows how to proxy requests going **to** grafana, not requests coming **from** grafana.

Reply:
Done.